internal/signature/validator.go

```diff
@@ -0,0 +1,105 @@
+package signature
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/DIMO-Network/token-exchange-api/internal/contracts/erc1271"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+var ERC1271MagicValue = [4]byte{0x16, 0x26, 0xba, 0x7e}
```

Comment:
I don't think this needs to be exported. Which removes the ability for other packages to change the value.
```suggestion
var erc1271MagicValue = [4]byte{0x16, 0x26, 0xba, 0x7e}
```

Reply:
I think I was using this var in another place before. Makese sense